test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -715,6 +715,34 @@ public void canReabsorbThirdMonster() {
 
   @Nested
   class CosmicBowlingBall {
+    @ParameterizedTest
+    @CsvSource({
+      // Off in the distance, you hear your cosmic bowling ball rattling around in the ball
+      // return system.
+      "1, 7, 8",
+      "1, 10, 9",
+      // You hear your cosmic bowling ball rattling around in the ball return system.
+      "2, 9, 7",
+      "2, 2, 4",
+      // You hear your cosmic bowling ball rattling around in the ball return system nearby.
+      "3, 5, 3",
+      "3, 1 ,2",
```

Comment:
minor, but your CSV is slightly off here
```suggestion
      "3, 1, 2",
```